File: api-clients/javascript-closure-angular/API/Client/Condition.js
goog.provide('API.Client.Condition');

/**
 * @record
 */
API.Client.Condition = function() {}

/**
 * Name of the condition.
 * @type {!string}
 * @export
 */
API.Client.Condition.prototype.name;

/**
 * Slug based on the condition name.
 * @type {!string}
 * @export
 */
API.Client.Condition.prototype.slug;

/**
 * Description of the condition.
 * @type {!string}
 * @export
 */
API.Client.Condition.prototype.description;

/**
 * Date and time record was created, UTC.
 * @type {!Date}
 * @export
 */
API.Client.Condition.prototype.createdAt;

/**
 * Date and time record was updated, UTC.
 * @type {!Date}
 * @export
 */
API.Client.Condition.prototype.updatedAt;

File: api-clients/async-scala/src/main/scala/io/swagger/client/model/Condition.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Condition (
  name: String,  // Name of the condition.
slug: Option[String],  // Slug based on the condition name.
description: Option[String],  // Description of the condition.
createdAt: Option[DateTime],  // Date and time record was created, UTC.
updatedAt: Option[DateTime]  // Date and time record was updated, UTC.
)
